packages/cms-data-sync/package.json

```diff
@@ -11,7 +11,7 @@
   "scripts": {
     "build:babel": "../../scripts/build-babel.sh build",
     "watch:babel": "../../scripts/build-babel.sh watch",
-    "migrate:news": "TS_NODE_COMPILER_OPTIONS='{\"module\": \"commonjs\"}' ts-node ./src/news/news.data-migration.ts",
+    "migrate:news": "TS_NODE_COMPILER_OPTIONS='{\"module\": \"commonjs\"}' ts-node ./src/index.ts",
```

Comment:
now we're not migrating only news in this script 😛 but if you want, we can rename it in another PR